src/main/java/nextstep/courses/service/SessionService.java

```diff
@@ -34,17 +34,15 @@ public void registerStudent(NsUser student, Long sessionId){
     }
 
     public void acceptStudent(NsUser teacher, Long sessionId, SelectedStudents students){
-        checkTeacher(teacher);
+        checkSessionAcceptAuth(teacher);
```

Comment:
```suggestion
        teacher.checkSessionAcceptAuth();
```

그냥 호출이라서 메소드 분리안해도 되지 않나요? ㅎ